core_lib/src/util/mathutils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef MATHUTILS_H
+#define MATHUTILS_H
+
+#define M_PI 3.14159265358979323846264338327950288
+
+struct MathUtils
```

Comment:
I like the idea of the mathutils, could probably be useful for some other things too. However, I think it would be better if we made it a namespace.

Reply:
Hmm alright we can do that ;)